sel4test-hw-matrix/steps.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I don't remember the details or how I got bitten by this, but there isn't always a `/bin/bash`. Don't really care, if the other scripts depend on it it's better to be consistent than fiddle for unknown future benefit.

Reply:
It's consistent with the other scripts. We do know there is a `/bin/bash` on the github runners (Ubuntu) and there is also one in the seL4 docker containers, so anything that runs in these should be safe.